### PR TITLE
Fix show readme.md in PyPi pages

### PR DESCRIPTION
**Pull request by drmcodes**
# Description

PyPi pages changed from plain hardcoded string to render README.md 

Fixes Issue: 

# Before Merge Checklist 
These items can be completed after PR is created.

(Check any items that are not applicable (NA) for this PR)

- [✅] JavaScript implementation
- [✅] Python implementation (NA if HTML beautifier)
- [✅] Added Tests to data file(s)
- [✅] Added command-line option(s) (NA if
- [✅] README.md documents new feature/option(s)

### CODE REVIEW

**Review by cclauss — APPROVED** [review]
Nice!

File: python/setup-css.py [hunk 1]
```diff
@@ -27,12 +27,16 @@ def run_tests(self):
...
+    with open("README.md", "r", encoding="utf-8") as fh:
+        long_description = fh.read()
+
 
 setup(
     name="cssbeautifier",
     version=__version__,
     description="CSS unobfuscator and beautifier.",
-    long_description=("Beautify, unpack or deobfuscate CSS"),
+    long_description=long_description,
```
Comment (cclauss) [inline python/setup-css.py:38]: Please revert lines 30-32 and then
```suggestion
    long_description=Path("README.md").read_text(),
```
and then at the top of the file add `from pathlib import Path`

File: python/setup-js.py [hunk 2]
```diff
@@ -27,15 +27,14 @@ def run_tests(self):
...
+        long_description = fh.read()
 setup(
     name="jsbeautifier",
     version=__version__,
     description="JavaScript unobfuscator and beautifier.",
-    long_description=(
-        "Beautify, unpack or deobfuscate JavaScript. "
-        "Handles popular online obfuscators."
-    ),
+    long_description=long_description,
```
Comment (cclauss) [inline python/setup-js.py:36]: Same as above.

**Review by cclauss — CHANGES_REQUESTED** [review]

File: python/setup-css.py (hunk 1 above)
Comment (drmcodes) [reply python/setup-css.py]: Ok thanks

File: python/setup-css.py [hunk 3]
```diff
@@ -27,12 +27,15 @@ def run_tests(self):
         errno = pytest.main(self.pytest_args)
         sys.exit(errno)
 
+    
+
```
Comment (cclauss) [inline python/setup-css.py:31]: Please remove these extra blank lines.  (There should be two blank lines, not four.)
```suggestion
```

File: python/setup-js.py [hunk 4]
```diff
@@ -27,15 +27,13 @@ def run_tests(self):
         errno = pytest.main(self.pytest_args)
         sys.exit(errno)
 
-
+
```
Comment (cclauss) [inline python/setup-js.py:30]:
```suggestion

```

**Review by cclauss — CHANGES_REQUESTED** [review]

File: python/setup-css.py (hunk 3 above)
Comment (drmcodes) [reply python/setup-css.py]: Sorry for the inconvenience, thanks for the feedback

File: python/setup-js.py (hunk 4 above)
Comment (cclauss) [reply python/setup-js.py:30]: Let's lose the whitespace.

**Review by cclauss — APPROVED** [review]

**Comment by cclauss** [comment]
This should inject the contents of https://github.com/beautifier/js-beautify/blob/main/README.md into the pages
* https://pypi.org/project/cssbeautifier
* https://pypi.org/project/jsbeautifier

**Review by bitwiseman — APPROVED** [review]

**Comment by bitwiseman** [comment]
@drmcodes resolve conflicts, please.

**Comment by cclauss** [comment]
I do not see any remaining conflicts.

**Comment by bitwiseman** [comment]
They resolved them.

**Comment by drmcodes** [comment]
Thx for the feedback.

**Comment by cclauss** [comment]
Why was this closed?

**Pull request closed without merging**